src/sentry/monitors/models.py

```diff
@@ -74,14 +74,24 @@ class MonitorEnvironmentValidationFailed(Exception):
     pass
 
 
+class MonitorObjectStatus(ObjectStatus):
+    WAITING = 4
+
+    @classmethod
+    def as_choices(cls) -> Sequence[Tuple[int, str]]:
+        choices = list(super().as_choices())
+        choices.append((cls.WAITING, "waiting"))
+        return tuple(choices)
```

Comment:
have to convert to `list` for typing
https://github.com/python/typeshed/issues/6985